modules/debugging/index.qmd

```diff
@@ -65,9 +65,9 @@ str(car)
 
 ```
 
-Oh no! An error message: <span style="color: red;">"Error: object 'car' not found"errors</span>!
+Oh no! An error message: ["Error: object 'car' not found"]{style="color: red;"}!
```

Comment:
With using Penguins, would it be best to put it in the yaml `packages: ` part of the qmd?
I'm noticing if I try use it to replace cars, it's a bit wonky because unlike cars when I render it and have it in the yaml  - it wont give the errors, but if I don't have it in that part loading the package wont work because it has to be installed. Maybe I can readjust the example to the user typing in 'palmerpenguins' instead of 'penguins' and getting an error?

Reply:
@mitchelloharawild Hey Mitch! Was wondering if I could get guidance with this ☺️ I'm trying to use `palmerpenguins` instead of `mtcars` however if I load it into the YAML packages section I can't get the errors I need (i.e. object 'penguins' not found). If I try load the library in the chunk like:
```library(palmerpenguins)
str(<<penguins>>)
  
???
  
p <- .code[[1]][[2]][[2]]
  
c("did you put in penguins?" = !"penguins" %in% all.vars(.code))
```

I get the error **Error: there is no package called ‘palmerpenguins’.**  But if I include in the chunk

```
install.packages("palmerpenguins")
library(palmerpenguins)
str(<<penguins>>)
  
???
  
p <- .code[[1]][[2]][[2]]
  
c("did you put in penguins?" = !"penguins" %in% all.vars(.code))
```

I get the error: **Error: This version of R is not set up to install source packages
If it was installed from an RPM, you may need the R-devel RPM**

What is the best way to approach this?